### DISCUSSION TITLE
"errorCode":"error.security.api-key-invalid"

### DISCUSSION THREAD

**Discussion by EdwardThJones** [ig-python/ig-markets-api-python-library#235] [Q&A]
I've been using trading_ig with a DEMO account and it worked fine.  This week I tried switching to a LIVE environment but I get the following error message:

"errorCode":"error.security.api-key-invalid"

I have tested my credentials on the IG API Companion and they work in the PROD/DEAL environment, therefore I am happy there is no issue with my log-on details or API key.  I'd be grateful for any suggestions you may have on how to solve this problem.

The issue has now been resolved.

**Comment by Zartexo** [comment]
Hi @EdwardThJones ,

i have the same issue (Check open issues). May I know how you solved the issue?

Best regards
